osquery/tables/system/linux/os_version.cpp

```diff
@@ -22,7 +22,13 @@ namespace xp = boost::xpressive;
 namespace osquery {
 namespace tables {
 
-#if defined(REDHAT_BASED)
+#if defined(RHEL)
+const std::string kLinuxOSRelease = "/etc/redhat-release";
+const std::string kLinuxOSRegex =
+    "(?P<name>[\\w+\\s]+) .* "
+    "(?P<major>[0-9]+)\\.(?P<minor>[0-9]+) "
+    "(?P<patch>\\(\\w+\\))";
+#elif defined(CENTOS)
```

Comment:
Actually, it looks like CentOS doesn't work either.  Can we get rid of the if statement all togehter? 

```
$ cat /etc/redhat-release
CentOS release 6.7 (Final
```

```
osquery> select * from os_version;
+------+-------+-------+-------+-------+
| name | major | minor | patch | build |
+------+-------+-------+-------+-------+
|      | -1    | -1    | -1    |       |
+------+-------+-------+-------+-------+
```

```
[vagrant@localhost ~]$ rpm -qi osquery
Name        : osquery                      Relocations: /
Version     : 1.5.2_57_g65162e7                 Vendor: Facebook
Release     : 1.el6                         Build Date: Tue 15 Sep 2015 01:47:48 AM UTC
...
```